app/styles/front.scss

```diff
@@ -688,21 +690,51 @@ $autre-lighter: #cce4e7;
   cursor: help;
 }
 
-[data-toggle="buttons"] {
-  display: flex;
-  flex-direction: row;
+legend {
+  border-bottom: 0px;
```

Comment:
> If using the shorthand `border-<direction>` syntax, prefer using the `none` value rather than setting only the `border-width`; otherwise, it is expected that you're overriding just one property of the set, but you're currently disabling it altogether by setting its width to 0.